enclave-runtime/Cargo.lock

```diff
@@ -1160,17 +1170,41 @@ dependencies = [
  "digest 0.9.0",
 ]
 
+[[package]]
+name = "http"
+version = "0.2.1"
+source = "git+https://github.com/mesalock-linux/http-sgx?tag=sgx_1.1.3#62544a1833f98f1810a44877c5f1efe45f5327c0"
```

Comment:
We now have two http library imports. Since the http library from `source = "git+https://github.com/integritee-network/http-sgx?branch=sgx-experimental#307b5421fb7a489a114bede0dc05c8d32b804f49"` imports the newer bytes version, I'd go for the integritee-network library, if possible.

Reply:
thanks for noticing this, fixed now 👍